qa/tasks/parent_cache.py

```diff
@@ -0,0 +1,563 @@
+"""
+parent_cache task
+"""
+from cStringIO import StringIO
+
+import contextlib
+import logging
+import os
+import yaml
+import time
+
+from teuthology import misc as teuthology
+from teuthology import contextutil
+from tasks import rbd
+from teuthology.orchestra import run
+from teuthology.config import config as teuth_config
+
+from util.workunit import get_refspec_after_overrides
+
+log = logging.getLogger(__name__)
+
+DEFAULT_IMAGE_URL = 'http://cloud-images.ubuntu.com/releases/bionic/release/ubuntu-18.04-server-cloudimg-amd64.img'
```

Comment:
I think this task should only be concerned with configuring and running the daemon. Let other tasks like qemu just run on top of a cloned image.

Reply:
hi, jason, i have deleted original parent_cache.py file, and add new file called immutable_object_cache.py. Based on qemu.py function, this new file just be responsible for setup daemon, teardown daemon and thrashes daemon. 
thanks